### PR TITLE
Adding additional smart_proxy plugins to hourly cron sync of git repos

### PR DESCRIPTION
**Pull request by cpeters**
- smart_proxy_openscap
- smart-proxy-pulp

### CODE REVIEW

File: .openshift/cron/hourly/git_repos [hunk 1]
```diff
@@ -68,6 +68,8 @@ update_repo smart_proxy_discovery https://github.com/theforeman/smart_proxy_disc
 update_repo smart_proxy_discovery_image https://github.com/theforeman/smart_proxy_discovery_image
 update_repo smart_proxy_dynflow https://github.com/theforeman/smart_proxy_dynflow
 update_repo smart-proxy https://github.com/theforeman/smart-proxy.git
+update_repo smart_proxy_openscap https://github.com/theforeman/smart_proxy_openscap.git
+update_repo smart-proxy-pulp https://github.com/theforeman/smart-proxy-pulp.git
```
Comment (domcleal) [inline .openshift/cron/hourly/git_repos]: There's no corresponding project for this?

**Comment by cpeters** [comment]
@domcleal @ehelms 
After searching I wasn't able to find a corresponding project. I spoke with Eric and he suggested either creating projects for each or throwing them under the katello umbrella. Which is preferred?

**Comment by domcleal** [comment]
That should be up to whoever's maintaining it.

**Comment by cpeters** [comment]
Updated

**Pull request closed without merging**